src/Kris/LaravelFormBuilder/Fields/FormField.php

```diff
@@ -295,11 +295,13 @@ protected function prepareOptions(array $options = [])
 
             if ($this->parent->clientValidationEnabled()) {
                 $this->setOption('attr.required', 'required');
+            }
+        }
 
-                if ($parsedRules) {
-                    $attrs = $this->getOption('attr') + $parsedRules;
-                    $this->setOption('attr', $attrs);
-                }
+        if ($this->parent->clientValidationEnabled()) {
```

Comment:
You can move `$parsedRules` into this if to avoid double nesting:
```php
if ($this->parent->clientValidationEnabled() && $parsedRules) {
  $attrs = $this->getOption('attr') + $parsedRules;
  $this->setOption('attr', $attrs);
}
```